ct-app-unified/database/database_connection.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from core.components.parameters import Parameters
+from sqlalchemy import create_engine
+from sqlalchemy.engine.url import URL
+from sqlalchemy.orm import Session
+
+from .models import Base
+
+log = logging.getLogger()
+
+
+class DatabaseConnection:
+    """
+    Database connection class.
+
+    This class requires the following environment variables to be set:
+    - PGUSER: the database user name
+    - PGPASSWORD: the database user password
+    - PGHOST: the database host
+    - PGPORT: the database port
+    - PGDATABASE: the database name
+    - PGSSLMODE: the SSL mode
+    - PGSSLROOTCERT: the SSL root certificate
+    - PGSSLCERT: the SSL certificate
+    - PGSSLKEY: the SSL key
```

Comment:
It's better to assert this inside the init function than have a comment. Also easier to debug.

Reply:
Done in commit [5d257a2](https://github.com/hoprnet/ct-research/pull/409/commits/5d257a289cc2393a66dfcbfd5a7a7afd63325480)